backend/test/integration/GameStartEvents.integration.test.js

```diff
@@ -34,7 +39,17 @@ describe('GameStartEvents integration tests', () => {
             await connectAndJoin(clientSockets, i, port, lobbyCode);
         }
         /* eslint-enable no-await-in-loop */
-        await startGame(clientSockets);
+        await startGameSixPlayer(clientSockets);
```

Comment:
Can you combine the two startGame functions and pass through the number as a parameter?

Reply:
Good Idea. Ill make those changes.